src/common/emoji.ts

```diff
@@ -36,3 +36,10 @@ export function emojify(message: string) {
 		return emojiMap?.[code] || s;
 	});
 }
+
+export function getEmojis(): Record<string, string> {
+	if (emojiMap === undefined) {
+		return {};
+	}
+	return emojiMap;
+}
```

Comment:
Instead of having a separate `getEmojis`, lets just have `ensureEmojis` return the map. Then you don't need to make two calls when all you want is the map.